src/core/core.js

```diff
@@ -498,15 +497,13 @@ define(function (require) {
 
   p5.prototype.registerPreloadMethod = function(m) {
     p5.prototype._preloadMethods.push(m);
-  }.bind(this);
+  };
```

Comment:
Awesome, thanks! `this` in that instance ought to be pointing to the global, and since there's no usage of `this` *within* the functions there ought not to be any disruption—always good to double-check though.